Brocfile.js

```diff
@@ -33,20 +33,26 @@ var es5 = new Babel(lib, {
   blacklist: ['es6.modules']
 });
 
+function rollupConfig(entry) {
+  return new Rollup(es5, {
+    rollup: {
+      entry: 'lib/' + entry,
+      cache: 'BUILD:es6-promise',
```

Comment:
@azizhk haha no worries, I thought maybe you found some better caching strategy hidden deep inside rollup itself.